src/libxrpl/basics/contract.cpp

```diff
@@ -48,6 +49,7 @@ LogicError(std::string const& s) noexcept
 {
     JLOG(debugLog().fatal()) << s;
     std::cerr << "Logic error: " << s << std::endl;
+    UNREACHABLE("LogicError", {{"message", s}});
```

Comment:
```suggestion
    UNREACHABLE("ripple::detail::LogicError", {{"message", s}});
```

Reply:
I'd rather keep this one; it is special because there are different ways `LogicError` may happen, which is also why the extra parameter.